lib/sles4sap/sap_deployment_automation_framework/configure_workload_tfvars.pm

```diff
@@ -0,0 +1,393 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+package sles4sap::sap_deployment_automation_framework::configure_workload_tfvars;
+
+use strict;
+use warnings;
+use testapi;
+use Exporter qw(import);
+use Carp qw(croak);
+use utils qw(write_sut_file);
+use sles4sap::sap_deployment_automation_framework::deployment qw(get_os_variable get_fencing_mechanism);
+use sles4sap::sap_deployment_automation_framework::naming_conventions
+  qw(convert_region_to_short generate_resource_group_name);
+use sles4sap::sap_deployment_automation_framework::deployment_connector qw(find_deployment_id no_cleanup_tag);
+
+=head1 SYNOPSIS
+
+Library with common functions for Microsoft SDAF deployment automation that help with preparation of
+'WORKLOAD-ZONE' tfvars file.
+Generated file is following example template:
+https://github.com/Azure/SAP-automation-samples/blob/main/Terraform/WORKSPACES/SYSTEM/LAB-SECE-SAP04-L00/LAB-SECE-SAP04-L00.tfvars
+
+=cut
+
+our @EXPORT = qw(
+  create_workload_tfvars
+  compile_tfvars_section
```

Comment:
`compile_tfvars_section` does not seems to be used externally. Probably it does not need to be exported.